### ISSUE TITLE
Setting fontSize on legend label template truncates label too much

### ISSUE THREAD

**Issue by c-goetz** [amcharts/amcharts4#2063]
**Codepen:**
https://codepen.io/carlo_g/pen/vYEvMXK

**Description:**
Using a legend with `position="right"` sometimes truncates the label too much.

The attached pen demonstrates this behavior:
![image](https://user-images.githubusercontent.com/6413073/72898970-6113e700-3d25-11ea-975c-c673d7953302.png)

The first two items in the legend are truncated after the first letter.

Commenting line 85 out or setting the value to 16 fixes this:
![image](https://user-images.githubusercontent.com/6413073/72899073-90c2ef00-3d25-11ea-9733-41654f163a6e.png)

**Affected versions:**
Tested with 4.8.0 and 4.8.1

**Remarks:**
To me it seems that the error lies somewhere in `core/elements/Label.js` around line 441.
Changing this line from:
```
var maxChars = $math.max(elementText.length - excessChars - this.ellipsis.length, 1);
```
to:
```
var maxChars = $math.max(lineText.length - excessChars - this.ellipsis.length, 1);
```
fixes the issue for me, but I did not read the whole function.

Is this a bug or did I use some faulty configuration?

**Comment by martynasma** [comment]
Fixed in [4.8.2](https://github.com/amcharts/amcharts4/releases/tag/4.8.2).

## [4.8.2] - 2020-01-23

### Added
- `TreeMap.homeDataItem` getter added.

### Fixed
- Sometimes labels were not being truncated properly.
- `ValueAxis` with values <1 might not display labels (since `4.8.1` only).
- Axis syncing was not working on initial view if no animated theme was used.
- Sometimes undefined variables were being restored with default state where they shouldn't. (since `4.8.1` only).

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by c-goetz** [comment]
Works like a charm!
Thanks for the really fast fix.
Have a nice day!
